packages/pockethost.io/src/util/stores.ts

```diff
@@ -15,15 +15,16 @@ if (browser) {
    * Listen for auth change events. When we get at least one, the auth state is initialized.
    */
   onAuthChange((authStoreProps) => {
-    console.log(`onAuthChange in store`, { ...authStoreProps })
     authStoreState.set(authStoreProps)
     isAuthStateInitialized.set(true)
   })
 
   // Update derived stores when authStore changes
   authStoreState.subscribe((authStoreProps) => {
-    console.log(`subscriber change`, authStoreProps)
     isUserLoggedIn.set(authStoreProps.isValid)
     isUserVerified.set(!!authStoreProps.model?.verified)
   })
 }
+
+// Handles the intro modal for new users
+export const isFirstTimeRegistration = writable(true)
```

Comment:
I added a `isFirstTimeRegistrationCompleted` (default `false`) to the `users` collection. You should be able to do something now like:

```
onAuthChange( authStoreProps => {
   isFirstTimeRegistration.set(!authStoreProps.user.isFirrstTimeRegistrationCompleted)
})
```
Or something like that. You'll also probably need to add the field to the `User` record type and `authStoreProps` and add an updater to `PocketbaseClient`